### DISCUSSION TITLE
Uncaught TypeError: Cannot read properties of undefined (reading 'getPicture')

### DISCUSSION THREAD

**Discussion by glIrving** [quasarframework/quasar#13504] [CLI - Capacitor / Cordova modes]
when i use cordova-plugin-camera ,it print :**Uncaught TypeErrorr :Uncaught TypeError: Cannot read properties of undefined (reading 'getPicture')**, i need your help

**Comment by MilosPaunovic** [comment]
Unless you provide a feasible reproduction of some kind, don't expect anyone to be able to assist you, as this request is too vague; bunch of thigs could be the reason behind it.

My suggestion is to create a minimal reproduction repository and share it with us; if you find out that it is, in fact, Quasar related issue.

**Comment by glIrving** [comment]
thanks ，i think i have  resolved this issue， i guess that reason maybe refer to vite  ，when i changed to webpack and  make the version same it worked
